packages/commons/src/interop-token/models.ts

```diff
@@ -40,12 +40,21 @@ export type InteropToken = {
   serialized: string;
 };
 
+const Organization = z.union([
```

Comment:
since all fields we are not using are defined as `nullish`, is this union required?

Reply:
If I got it right here we need either a uuid of the organization or an object representing the organization.

Reply:
You are right, I really don't know why I thought it was like that 🤷‍♂️